src/index.ts

```diff
@@ -694,5 +702,29 @@ export const validateReceipt = async (
   }
 };
 
+/**
+ * Deeplinks to native interface that allows users to manage their subscriptions
+ * @param {string} [sku] Required for Android to locate specific subscription
+ * @returns {Promise<void>}
+ */
+export const deepLinkToSubscriptions = ({
+  sku,
```

Comment:
Would `skuAndroid` be better in this case? This follows one of the conventions in the repo for naming platform-specific props.

Reply:
Updated the parameter name from `sku` to `skuAndroid` to follow the repository's naming convention for platform-specific props. Thanks for the suggestion!